### DISCUSSION TITLE
[FEATURE]: Support ePost Global as shipping carrier

### DISCUSSION THREAD

**Discussion by sowsinsk** [etsy/open-api#720] [Ideas]
### Is there an existing ticket for this?

- [x] I have searched the existing tickets

### Describe the solution you'd like

Currently, [ePost Global](https://epostglobalshipping.com) is not supported as a shipping carrier. I would love to see ePost Global supported by Etsy. It would allow us to use the createReceiptShipment endpoint to fulfill orders complete with tracking information.

Thank you for considering!

**Comment by etsyachristensen** [comment]
@sowsinsk In order to support new shipping carriers, we have to have some specific information. Often it's easiest to have the carrier contact us to discuss adding them. But lacking that, we'd have to get the following information:

1. Example URLs for several shipments. We need to see what the url structure looks like such that we can direct users to a common url with a tracking code. For example: `https://track.somecarrier.com/?tracking=xxxxxxxxx`

2. Example of some tracking codes (they can be fake, but must match the typical formatting of a tracking code for the carrier).

3. We use Aftership as a common tool for interfacing with many carriers. If they already work with Aftership it should be simple enough to add them to our system.

I do see that they work with ShipStation. I believe that ShipStation is one of our existing carrier integrations. You may want to consider using ShipStation for a quicker resolution.

**Comment by sowsinsk** [comment]
@etsyachristensen I pulled together some information in hopes of [ePost Global](https://epostglobalshipping.com) being a supported carrier in the future. The tracking website was sent to me by our ePost Global rep. It is their simplified tracking-only site that was recently launched. They also have a tracking portal on their corporate site [here](https://portal.epgshipping.com/ParcelTracker/HomePageTracker), but it doesn't support embedded tracking codes in the URL. The site below does.

1. Example tracking URLs for shipments:

https://epgtrack.com/EPG250941300039727
https://epgtrack.com/EPG250941300039352
https://epgtrack.com/EPG250941300039260
https://epgtrack.com/EPG250941300039123

2. Here are some examples of valid tracking codes. I checked all our tracking codes over the past 2 years, and they all follow the same format: 3-letter prefix of "EPG" + 15 digits.

EPG250941300027229
EPG250941300022699
EPG250941300020282
EPG250941300018159

3. I checked, and ePost Global is supported by Aftership [here](https://www.aftership.com/couriers/epostglobal)

Thanks for considering this. If you need any other information, please let me know.

**Comment by etsyachristensen** [comment]
Sorry for the delay on this @sowsinsk. We are tracking this still. We're converting to an Idea discussion to continue tracking when we close down the Issues board.

**Comment by kbasali** [comment]
@etsyachristensen How often do you add new carriers? 

In Turkey, there is a carrier that tries to be added to the list but they have no luck with this. 

Personally, I have sent over 2k shipping with them and that would be great if Etsy will add them.

How can they contact you?

**Reply by etsyachristensen** [reply]
See comment above: https://github.com/etsy/open-api/discussions/720#discussioncomment-4393001

You can submit it to the Discussion board as a new idea.

**Comment by etsyachristensen** [comment]
@sowsinsk as it turns out this was already completed. Sorry about that. I didn't realize it had been handled.
